### PR TITLE
[TTS]Add slim for  TTS

### LINKED ISSUE #2682: 请问，fastspeech2 模型能否进行paddleslim 量化呢？声学模型部署，onnx 能否支持tensorrt 优化？

### PR DESCRIPTION
**Pull request by yt605155624**
use

paddleslim 2.4.0 
paddlepaddle develop (not merged yet) https://github.com/PaddlePaddle/Paddle/pull/48912

fix https://github.com/PaddlePaddle/PaddleSpeech/issues/2682

### CODE REVIEW

File: paddlespeech/t2s/exps/PTQ_static.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        seed = 111
+        np.random.seed(seed)
+        paddle.seed(seed)
+        random.seed(seed)
+        shuffle = False
+
+    place = paddle.CUDAPlace(0) if args.ngpu > 0 else paddle.CPUPlace()
+    with jsonlines.open(args.dev_metadata, 'r') as reader:
+        dev_metadata = list(reader)
+    # dev_metadata = dev_metadata[:args.batch_size]
```
Comment (lym0302) [inline paddlespeech/t2s/exps/PTQ_static.py]: 是不是要删掉

File: paddlespeech/t2s/exps/PTQ_static.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+            'pwgan_csmsc',
+            'pwgan_aishell3',
+            'pwgan_ljspeech',
+            'pwgan_vctk',
+            'mb_melgan_csmsc',
+            'hifigan_csmsc',
+            'hifigan_aishell3',
+            'hifigan_ljspeech',
+            'hifigan_vctk',
+            'wavernn_csmsc',
```
Comment (lym0302) [inline paddlespeech/t2s/exps/PTQ_static.py]: tacotron2 和 wavernn 也OK吗

**Review by lym0302 — DISMISSED** [review]

File: paddlespeech/t2s/exps/PTQ_static.py (hunk 1 above)
Comment (yt605155624) [reply paddlespeech/t2s/exps/PTQ_static.py]: done

File: paddlespeech/t2s/exps/PTQ_static.py (hunk 2 above)
Comment (yt605155624) [reply paddlespeech/t2s/exps/PTQ_static.py]: done

**Review by lym0302 — APPROVED** [review]

**Pull request merged**